bin/process.py

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env python
```

Comment:
On my system, Python executable files under `/usr/bin` look like this:
```sh
$ ls -l /usr/bin | grep python
...
python -> python3
python2 -> python2.7
python2.7
python3 -> python3.4
python3.4
...
```
I think that it does on all systems. Depending on your system maintainers choice, python may point to `python2` or `python3` which in turn points to the specific python version executable for instance, `python2->python2.7`. 

That's why I think that if the program runs on python 3, this line should be:
```python
#!/usr/bin/env python3
```
Then calling the script with it's path would always lead to calling `python3`. Do you think there's any drawback doing this?